src/App.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { BrowserRouter as Router, Switch } from 'react-router-dom';
 
 import './styles/app.scss';
+import PublicRoute from './components/common/PublicRoute';
+import PrivateRoute from './components/common/PrivateRoute';
+import HomePage from './containers/HomePage';
 import LoginPage from './containers/LoginPage';
 
 const App = () => (
   <Router>
-    <div>
-      <Switch>
-        <Route path="/" component={LoginPage} />
-        <Route path="/login" component={LoginPage} />
-      </Switch>
-    </div>
+    <Switch>
+      <PublicRoute path="/login" exact component={LoginPage} />
```

Comment:
didn't you have these routes already defined in a routes object? Use them from there

Reply:
Yes, I forgot to use them here 😅